service/history/timerQueueProcessor.go

```diff
@@ -209,6 +212,15 @@ func (t *timerQueueProcessorImpl) NotifyNewTimer(timerTasks []persistence.Task)
 			t.minPendingTimer = ts
 			updatedMinTimer = true
 		}
+
+		switch task.GetType() {
+		case persistence.TaskTypeDecisionTimeout:
+			t.metricsClient.IncCounter(metrics.TimerTaskDecisionTimeoutScope, metrics.NewCounter)
```

Comment:
Because unlike the timeout scopes, these are *not* timeouts, but rather request to create new timers